### ISSUE TITLE
My jetpack: Replace Extras card with new Monitor Card

### ISSUE THREAD

**Issue by oskosk** [Automattic/jetpack#23516]
Depends on #[#23517](https://github.com/Automattic/jetpack/issues/23517)
> We’re proposing Monitor bubbles to the top, as its one of our most beloved and popular features.

![image](https://user-images.githubusercontent.com/746152/159260952-1b9ec00a-edd8-4ba8-a073-ab145c30ad52.png)

**Comment by oskosk** [comment]
Closing as won't do. 

See p1HpG7-fbl-p2#comment-53134 for context
